packages/expo-router/src/utils/splash.ts

```diff
@@ -0,0 +1,39 @@
+import { requireOptionalNativeModule } from 'expo';
```

Comment:
i wasn't sure if it was needed to create **utils/splash.ts** in addition to **views/Splash.tsx** because i didn't understand the reasoning behind why we imported the splash methods from **views/Splash.tsx** in some cases but **expo-splash-screen** in others. so, to preserve the same behavior with respect to `'use client'` i replaced imports from `expo-splash-screen` to be from `utils/splash.ts` and left the view imports as-is

Reply:
@EvanBacon - can you let me know if this is the right thing to do here?